roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -24,30 +24,21 @@ def capsule_lce_args(action, capsule_id, env)
   "--csv capsule content #{action}-lifecycle-environment --id #{capsule_id} --lifecycle-environment-id #{env}"
 end
 
-external_capsules = []
 external_capsule_ids = get_info_from_hammer("--csv capsule list --search 'feature = \"Pulp Node\"'")
```

Comment:
Unrelated, but this will fail on 6.10 where the `Pulp Node` feature is no longer present.

Reply:
At the time this runs, the hostname changeover has already been performed. So I think this can be solved with:

```ruby
require 'socket'
internal_capsule = Socket.gethostname
external_capsule_lifecycle_environments = get_info_from_hammer("--csv capsule list --search 'name != \"#{internal_capsule}\"'").split("\n").map do |id|
  {
    ...
  }
end
```

I am not sure if it's possible that the internal capsule could be renamed to no longer match Satellite's hostname.

Searching by id is not an option unfortunately:

```shell
# hammer --csv capsule list --search 'id != 1'
400 Bad Request
  Field 'id' not recognized for searching!
```